resource/http.go

```diff
@@ -9,36 +10,52 @@ import (
 
 type HTTP struct {
 	Title             string   `json:"title,omitempty" yaml:"title,omitempty"`
+	URL               string   `json:"url,omitempty" yaml:"url,omitempty"`
 	Meta              meta     `json:"meta,omitempty" yaml:"meta,omitempty"`
 	HTTP              string   `json:"-" yaml:"-"`
+	Method            string   `json:"method,omitempty" yaml:"method,omitempty"`
 	Status            matcher  `json:"status" yaml:"status"`
 	AllowInsecure     bool     `json:"allow-insecure" yaml:"allow-insecure"`
 	NoFollowRedirects bool     `json:"no-follow-redirects" yaml:"no-follow-redirects"`
 	Timeout           int      `json:"timeout" yaml:"timeout"`
 	RequestHeader     []string `json:"request-headers,omitempty" yaml:"request-headers,omitempty"`
+	RequestBody       string   `json:"request-bod,omitemptyy" yaml:"request-body,omitempty"`
 	Headers           []string `json:"headers,omitempty" yaml:"headers,omitempty"`
 	Body              []string `json:"body" yaml:"body"`
 	Username          string   `json:"username,omitempty" yaml:"username,omitempty"`
 	Password          string   `json:"password,omitempty" yaml:"password,omitempty"`
 	Skip              bool     `json:"skip,omitempty" yaml:"skip,omitempty"`
 }
 
-func (u *HTTP) ID() string      { return u.HTTP }
+func (u *HTTP) ID() string {
+	if u.URL != "" && u.URL != u.HTTP {
+		return fmt.Sprintf("%s: %s", u.HTTP, u.URL)
```

Comment:
This makes the output inconsistent with command. Let's exclude the URL for now when `url` is specified .

I'm open to discussing this further, feel free to open up an issue dedicated to this if you feel strongly about it one way or another.

Just opting for consistency then we can follow up with a PR that changes that to another consistent pattern.

Reply:
Oh, didn't take a look at the current source code and based mine on #434 changes. Modifying it